packages/instant/src/util/util.ts

```diff
@@ -2,4 +2,5 @@ import * as _ from 'lodash';
 
 export const util = {
     boundNoop: _.noop.bind(_),
+    createHrefOnClick: (href: string) => () => window.open(href, '_blank'),
```

Comment:
nit but this feels like a weird name: `createHrefOnClick` makes me think it will generate an `a` tag or something.  Really this is just something like `openUrlInNewWindow`, and `url` feels like a more natural parameter name than `href` here